orchestration/api/instances.py

```diff
@@ -11,3 +11,92 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from flask import jsonify
+from flask import Blueprint
+from orchestration.connectionmanager.connector import Connector
+from flask import request
+import json
+from orchestration.db.api \
+    import create_workflow, list_workflows
+from orchestration.api.apiconstants import Apiconstants
+
+instance = Blueprint("instance", __name__)
+
+# API to execute a service.
+# For example, St2 execution of this API will require 'action'
+# as one of the parameter. 'parameters' maybe required for the Actions
+# St2 example of data:/v1beta/orchestration/instances -H
+# "Content-Type:application/json" -d
+# '{"action":"aks.ak_echo_py", "parameters":{"message":"Hello World"}}'
+@instance.route("/v1beta/orchestration/instances", methods=['POST'])
+def instance_ops():
+    c = Connector().morph()
+    content = request.get_json()
+    rc, ret = c.execute_action(content)
+    if(rc != Apiconstants.HTTP_OK):
+        return jsonify(response=json.loads(ret)), rc
+
+    ret_json = json.loads(ret)
+    wf_hash = {}
+    wf_hash['id'] = ret_json['id']
+    wf_hash['name'] = ret_json['action']['name']
+    wf_hash['input'] = json.dumps(ret_json['parameters'])
+    wf_hash['workflow_definition_id'] = ret_json['action']['ref']
+
+    # Create the record of this instance in DB
+    create_workflow(None, wf_hash)
+    return jsonify(response=ret_json), 200
+
+
+@instance.route("/v1beta/orchestration/instances/create", methods=['POST'])
```

Comment:
Can we remove the create from the endpoint

Reply:
Will do that. This api is redundant as of now as we are registering actions through CLI.
Right now code doesn’t get called

Reply:
We can open issue and fix it.